api/oc_swupdate.c

```diff
@@ -455,7 +467,7 @@ oc_swupdate_decode(oc_rep_t *rep, size_t device)
 }
 
 /**
- * post method for "/sw" resource.
```

Comment:
why the oic part?
that is not needed, it makes it just a longer url

Reply:
The software update resource is implemented as a device configuration resource (DCR). The URLs of most other DCRs are prefixed with `oic`. So I thought it's a kind of grouping for DCRs that iotivity-lite does and I propose to adjust the software update resource to follow that scheme.

It is not necessary, I just think it makes the DCR URLs more consistent.

Reply:
I want to mentioned that we have resource `/oc/con`  and it has  comment that it cannot be changed: https://github.com/iotivity/iotivity-lite/blob/master/api/oc_core_res.c#L54

Reply:
why change it?
this does not make the stack more consistent... 
the /oic/res etc are fixed by the spec the /sw is not.. so do not use /oic/ as a prefix, it would hint that that is a url specified by the spec

Reply:
> the /oic/res etc are fixed by the spec the /sw is not.. so do not use /oic/ as a prefix, it would hint that that is a url specified by the spec

Thanks for the explanation @WAvdBeek. I agree that `/oic/` is the wrong prefix.

But I would still prefer to change it to a `/oc` prefix: In my opinion, all resources that the iotivity-lite stack implements with a hard-coded URL should have the common URL prefix `/oc` (like the `/oc/con` or `/oc/wk/introspection` resources).  That's similar to the best practice in REST API design where the resources are grouped with such prefixes (that's probably where my preference comes from).

And what about the resource name `sw`? To me that acronym just stands for "software". `swu` or even `swupd` (as the spec names as a example) would be more expressive, in my opinion.

Reply:
oc/swu is fine by me

Reply:
Changed the URL to `/oc/swu`.